src/main/java/com/zufar/onlinestore/payment/entity/Payment.java

```diff
@@ -25,15 +30,38 @@ public class Payment {
     @Column(name = "payment_id")
     private Long paymentId;
 
+    @Column(nullable = false, unique = true, name = "payment_intent_id")
```

Comment:
please,
name then unique then nullable
What do you think if we follow this priority order of @Column annotation attributes?
If you agree, fix it everywhere, please

Reply:
fix it please

Reply:
`+`